xml/System.Windows.Forms/Control.xml

```diff
@@ -15138,7 +15142,11 @@ MyControl.Font = New Font(MyControl.Font, _
         <param name="msg">A <see cref="T:System.Windows.Forms.Message" /> that represents the message to process.</param>
         <summary>Preprocesses keyboard or input messages within the message loop before they are dispatched.</summary>
         <returns>One of the <see cref="T:System.Windows.Forms.PreProcessControlState" /> values, depending on whether <see cref="M:System.Windows.Forms.Control.PreProcessMessage(System.Windows.Forms.Message@)" /> is <see langword="true" /> or <see langword="false" /> and whether <see cref="M:System.Windows.Forms.Control.IsInputKey(System.Windows.Forms.Keys)" /> or <see cref="M:System.Windows.Forms.Control.IsInputChar(System.Char)" /> are <see langword="true" /> or <see langword="false" />.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <para> This is the method that is called directly by the <see cref="T:System.Windows.Forms.Application" />'s message loop.
+              See <see cref="M:System.Windows.Forms.Application.ThreadContext.PreTranslateMessage(Interop.User32.MSG@)" />.
```

Comment:
Nit: Could add this is a \<seealso> instead.

Reply:
Removing in the suggestion below.